WowPacketParser/Enums/UpdateFields.cs

```diff
@@ -792,16 +792,19 @@ public enum CorpseDynamicField
 
     public enum AreaTriggerField
```

Comment:
Good question, only did this modification to add missing fields like AREATRIGGER_DECAL_PROPERTIES_ID